.github/workflows/lint.yml

```diff
@@ -0,0 +1,28 @@
+name: Tool linting
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+      - name: Perform linting
+        run: make pr_check
```

Comment:
do you think we should also run `make lint`, it is supposed to check that the categories are correct.

Reply:
It looks like `make lint` is just updating the lock files? I'm not sure this should be run in the PR.

Reply:
Its also checking against a schema.yml file if all categories are actually defined.